example/src/main/kotlin/com/chutneytesting/example/scenario/http_scenario.kt

```diff
@@ -0,0 +1,77 @@
+package com.chutneytesting.example.scenario
+
+import com.chutneytesting.kotlin.dsl.*
+
+const val HTTP_TARGET_NAME = "HTTP_TARGET"
+
+const val FILMS_ENDPOINT = "/films"
+
+var FILM = """
+                {
+                "title": "Castle in the Sky",
+                "director": "Hayao Miyazaki",
+                "rating": "%rating%",
+                "category": "fiction"
+                }
+            """
+
+
+val http_scenario = Scenario(title = "Films library") {
+    Given("I save a new film") {
+        HttpPostAction(
+            target = HTTP_TARGET_NAME,
+            uri = FILMS_ENDPOINT,
+            body = FILM.trimIndent(),
+            headers = mapOf(
+                "Content-Type" to "application/json"
+            ),
+            validations = mapOf(
+                "creation_status" to "#status.equals(201)".elEval()
+            ),
+            outputs = mapOf(
+                "filmId" to "#body".elEval()
+            )
+        )
+    }
+
+    When ("I update rating") {
+        HttpPatchAction(
+            target = HTTP_TARGET_NAME,
+            uri = "$FILMS_ENDPOINT/\${#filmId}",
+            body = """
+                {
+                "rating": "79",
+                }
+            """.trimIndent(),
+            headers = mapOf(
+                "Content-Type" to "application/json"
+            ),
+            validations = mapOf(
+                "update_status" to "#status.equals(200)".elEval()
+            )
+        )
+    }
+
+    Then ("I check that rating was updated") {
+        Step("I get film by id") {
+            HttpGetAction(
+                target = HTTP_TARGET_NAME,
+                uri = "$FILMS_ENDPOINT/\${#filmId}",
+                headers = mapOf(
+                    "Content-Type" to "application/json"
+                ),
+                validations = mapOf(
+                    "get_status" to "#status.equals(200)".elEval()
```

Comment:
you could factorize it. It could be done in a commonHttp file by example

Reply:
Moreover, the validation key would be better worded like `status_ok` or `status_200` rather than `get_status`